src/pydata_sphinx_theme/assets/styles/base/_base.scss

```diff
@@ -71,46 +71,52 @@ a {
   }
 }
 
-.heading-style {
+%heading-style {
```

Comment:
Nobody's using this class directly are they?

Reply:
By nobody I meant adopters of the theme. Seems very unlikely to me that any adopters of PyData Theme are writing directives like:

```rst
.. class:: heading-style

Pretend I'm a heading
```

But I wanted to check that assumption.

Reply:
I just did a GitHub code search for ".. class:: heading-style" and ":class: heading-style" and it gave me nothing, so I'm going to consider this thread resolved.